modules/local/qiime2_alphararefaction.nf

```diff
@@ -1,7 +1,8 @@
 process QIIME2_ALPHARAREFACTION {
     label 'process_low'
 
-    container "qiime2/core:2023.7"
+    conda "${projectDir}/modules/local/envs/qiime2-amplicon-2024.10-py310-linux-conda.yml"
+    container "qiime2/amplicon:2024.10"
```

Comment:
Is this the preferred way? In metatdenovo we transitioned to having the subdirectory with `main.nf` and `environment.yml` just like with the official modules. (That would be a lot of renaming and duplication of yml files in this case of course.)